build_for_ultimaker.sh

```diff
@@ -36,6 +36,7 @@ run_in_docker()
 {
     docker run \
         --rm \
+        -it \
```

Comment:
I think this is a mistake?

Reply:
Nope, you stated that the `menuconfig` option wasn't working, since it triggers an interactive session, we need Docker to accept input.

Reply:
Ah i see know.